packages/e2e-tests/src/specs/wordpress/postLocking.js

```diff
@@ -34,8 +33,6 @@ const percyCSS = `.dashboard-grid-item-date { display: none; }`;
 const storyTitle = 'Test post lock';
 
 describe('Post Locking', () => {
-  withExperimentalFeatures(['enablePostLocking']);
```

Comment:
This does not need to be removed.

If the experiment is already enabled by default, then `withExperimentalFeatures` won't do anything.

This only needs to be removed once we completely remove all feature flag code.

Reply:
Fixed.